seatunnel-connectors-v2/connector-jdbc/src/main/java/org/apache/seatunnel/connectors/seatunnel/jdbc/catalog/dm/DamengCatalog.java

```diff
@@ -184,4 +169,60 @@ public CatalogTable getTable(String sqlQuery) throws SQLException {
         Connection defaultConnection = getConnection(defaultUrl);
         return CatalogUtils.getCatalogTable(defaultConnection, sqlQuery, new DmdbTypeMapper());
     }
+
+    @Override
+    public CatalogTable getTable(TablePath tablePath)
```

Comment:
any difference with https://github.com/apache/seatunnel/blob/57f47c2064473af310cc68eafed5fbbab1d03112/seatunnel-connectors-v2/connector-jdbc/src/main/java/org/apache/seatunnel/connectors/seatunnel/jdbc/catalog/AbstractJdbcCatalog.java#L164 ?

Reply:
The Super method used the sql to get the column medata data.
![image](https://github.com/user-attachments/assets/9bbcd4c0-8836-47f6-94eb-83f66f85583e)

In dm database, if you want to achieve this function, you usually have to write a very complex sql which is not convenient. Therefore, I directly used the corresponding method of the jdbc driver to get the column information, which is accurate and easy to implement

Reply:
I will abstract the code, it does look weird, as if the code repeats itself